### PR TITLE
Apprentice Commitments Confirm Your Training Provider

### PR DESCRIPTION
**Pull request by benagain**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/AddApprenticeshipSteps.cs [hunk 1]
```diff
@@ -113,6 +113,7 @@ public void ThenTheRequestToTheInnerApiWasMappedCorrectly()
             innerApiRequest.ApprenticeshipId.Should().Be(_request.ApprenticeshipId);
             innerApiRequest.EmployerName.Should().Be(_request.EmployerName);
             innerApiRequest.EmployerAccountLegalEntityId.Should().Be(_request.EmployerAccountLegalEntityId);
+            innerApiRequest.TrainingProviderName.Should().Be("Provisional Training Provider Name");
```
Comment (Pauljgraham) [inline src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/AddApprenticeshipSteps.cs:116]: Just wondering if TrainingProviderId is supposed to be here?

File: src/SFA.DAS.ApprenticeCommitments/Apis/InnerApi/CreateApprenticeshipRequest.cs [hunk 2]
```diff
@@ -17,6 +17,7 @@ public class CreateApprenticeshipRequestData
         public string Email { get; set; }
         public string EmployerName { get; set; }
         public long EmployerAccountLegalEntityId { get; set; }
+        public string TrainingProviderName { get; set; }
```
Comment (Pauljgraham) [inline src/SFA.DAS.ApprenticeCommitments/Apis/InnerApi/CreateApprenticeshipRequest.cs:20]: Definitely TrainingProviderId  should be here

File: src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs [hunk 3]
```diff
@@ -40,7 +40,8 @@ public class CreateApprenticeshipCommandHandler : IRequestHandler<CreateApprenti
                 ApprenticeshipId = command.ApprenticeshipId,
                 Email = command.Email,
                 EmployerName = command.EmployerName,
-                EmployerAccountLegalEntityId = command.EmployerAccountLegalEntityId
+                EmployerAccountLegalEntityId = command.EmployerAccountLegalEntityId,
+                TrainingProviderName = "Provisional Training Provider Name",
             });
```
Comment (Pauljgraham) [inline src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs:45]: I'm wondering where are we getting the TrainingProviderId from. We actually have a choice of getting it from the event or from the call to commitments.

**Review by Pauljgraham — CHANGES_REQUESTED** [review]
Think we are missing the TrainingProviderId

File: src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs (hunk 3 above)
Comment (benagain) [reply src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs:45]: That will be part of CS-286

File: src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/AddApprenticeshipSteps.cs (hunk 1 above)
Comment (benagain) [reply src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/AddApprenticeshipSteps.cs:116]: Yep, I've added it in with a dummy value like the name.

File: src/SFA.DAS.ApprenticeCommitments/Apis/InnerApi/CreateApprenticeshipRequest.cs (hunk 2 above)
Comment (benagain) [reply src/SFA.DAS.ApprenticeCommitments/Apis/InnerApi/CreateApprenticeshipRequest.cs:20]: Yep, I've added it in with a dummy value just like the name.

**Review by Pauljgraham — APPROVED** [review]

**Pull request merged**